src/seedsigner/views/scan_views.py

```diff
@@ -102,6 +102,9 @@ def run(self):
             else:
                 return Destination(NotYetImplementedView)
 
+        if self.decoder.is_invalid:
+            raise Exception("QRCode not recognized or not yet supported.")
+
```

Comment:
Added better note for when an unrecognized QR is scanned. Currently is simply goes to home without any message.

Reply:
I'm a fan of this.

I frequently run into this.  In every case so far it's been user error... but I wasn't sure if it might be some new bug I've introduced while testing elsewhere, or if I'm indeed scanning the wrong qrcode (most recently, trying to load a single seed's xpub in Sparrow instead of the wallet descriptor xpub just above).   I cannot be the only one that makes this mistake frequently and suspects "bug in seedsigner" when "bug in my head" is the better diagnosis.  This exception will lead me at least, to question what I'm doing instead of retrying the same mistake and hoping for better results.

Reply:
💯

Always nice to have clearer messages.

Reply:
TY @EverydayBitcoiner, this improvement has been borrowed (as an elif at same level) in pr #323